src/devices/video/fixfreq.h

```diff
@@ -2,60 +2,78 @@
 // copyright-holders:Couriersud
 /***************************************************************************
 
-    fixfreq.h
+	fixfreq.h
 
-    Fixed frequency monochrome monitor emulation
+	Fixed frequency monochrome monitor emulation
 
-    The driver is intended for drivers which provide an analog video signal.
-    VSYNC and HSYNC levels are used to create the bitmap.
+	The driver is intended for drivers which provide an analog video signal.
+	VSYNC and HSYNC levels are used to create the bitmap.
 
 ***************************************************************************/
 
 #ifndef MAME_VIDEO_FIXFREQ_H
 #define MAME_VIDEO_FIXFREQ_H
 
+#include "emu.h"
```

Comment:
You're not supposed to include `emu.h` from other headers.

Reply:
This implies that I have to include `emu.h` in `fixfreq.cpp` before I include `fixfreq.h`. 
That also means that header files will fail standalone compilation. 
Just because I am curious - Is there any specific reason for this rule?

Reply:
All checks passed, applied `emu.h` suggestion.

Reply:
I'm not exactly sure what the reasons are or if they remain sound, but I suspect they have to do with the fact that `emu.h` is built as a precompiled header.

Reply:
Ok, thanks, rings a bell. Years ago the include order mattered. Can't remember why.  The "rule" just does not fit to textbook-best-practice as found in a lot of places. Same for the fact that `emu.h` includes `<list>`, `<vector>`  and others before the other mame includes. Basically it is a one-size-fits-all approach which reads dozens of include files.